oas

```diff
@@ -142,6 +142,65 @@ paths:
               schema:
                 $ref: '#/components/schemas/Animal'
 
+  /events:
+    get:
+      summary: Get events for the animals
+      tags:
+        - Animal Information
+      parameters:
+        - name: date_start
+          in: query
+          required: true
+          schema: 
+            type: string
+            format: date-time
+        - name: date_end
+          in: query
+          description: default is NOW
+          schema: 
+            type: string
+            format: date-time
+        - name: animal_urn
+          in: query
+          schema: 
+            type: string
+        - name: category
+          in: query
+          schema: 
+            $ref: '#/components/schemas/EventCategory'
+      responses:
+        200:
+          description: 'successful operation'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Event'
+  /weights:
+    get:
+      tags:
+        - Animal Information
+      summary: Get weights for a specific date
```

Comment:
Of all animals over an arbitrarily large time range? Could this not result in a problematically massive data set?

Reply:
no, this parameter allows to choose one specific date. That should be possible even for big farms.